source/ui.spec.ts

```diff
@@ -55,7 +55,7 @@ describe('source/ui.ts', () => {
   describe('UIManager', () => {
     let uiManager: UIManager;
     const routeManager = new RouteManager();
-    const proxyManager = new ProxyManager([], routeManager);
+    const proxyManager = new ProxyManager(routeManager, []);
```

Comment:
The second param has a default value, can we omit it here?

Reply:
For sure.